docs/staff-guide/chat.md

```diff
@@ -1,3 +1,7 @@
 # Chat
 
-/chatlock
\ No newline at end of file
+Chat can be locked and unlocked with `/chatlock`.
+
+[//]: # (I'm just assuming it can be unlocked with the same command here)
+
+To check past chat messages, use [core protect's logs.](/docs/staff-guide/core.md/#lookup)
```

Comment:
```suggestion
To check past chat messages, use [core protect's logs.](/staff-guide/core#lookup)
```

Reply:
`/staff-guide/core/#lookup` actually works, but I'll change it for the sake of consistency

Reply:
CoreProtect chat lookup does work (which surprised me a little), but it only shows messages sent on the survival server and it's much easier to use Discord for this (easy to scroll, has search etc).

Reply:
Oh shoot, true. I'm a boomer so before discord that's what I used for chat logs. And it's interesting that it doesn't log messages from the creative server, though its not surprising.